lib/twingly/url_cache.rb

```diff
@@ -47,5 +55,13 @@ def options
     def servers
       ENV.fetch("MEMCACHIER_SERVERS") { "localhost" }.split(",")
     end
+
+    def with_memcached_exception_handling(&block)
```

Comment:
I feel the method name hides the fact that retries are part of the mix. Also, the method seems to be doing two separate things, perhaps revise into a two step rocket?

Reply:
Isn't the retries is part of the exception handling? :)

Moving the retries to a separate method might make things a little more clear though.

Reply:
IMHO, the two separate things that are going on here are:

* Retrying
* Converting exception types

Also, I withhold that looking at the method name you can't really figure out that the method will retry. One could pass a retry parameter to make that part more clear though.

Reply:
Changed the name of the method in b8f06a3 and also made a separate method of the "retry" part to make it a bit easier to understand what happens.